server/src/pages/shot/editor.js

```diff
@@ -100,18 +100,20 @@ exports.Editor = class Editor extends React.Component {
     this.state = {
       tool: 'pen',
       color: '#000',
-      size: '5'
+      size: '5',
+      saveDisabled: true
     };
   }
 
   render() {
+    let color = this.state.color == "rgb(255, 255, 255)" || this.state.color == "#FFF" ? "white" : null;
```

Comment:
This code is present several places, and isn't really complete (e.g., `#fff`). It might be fine that it isn't complete, but it should be in its own function (i.e., `isColorWhite(color)`) in case in the future it does need to be more complete